ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Thank you for contributing to PaddlePaddle. Submitting an issue is a great help for us.
+Both Chinese and English issues are welcome.
+
+Before you submitting the issue, there are several things should be check.
+
+- [ ] Was there a similar issue submitted or resolved before? You could search issue in the github.
+- [ ] Did you go to the search engine for your question?
+- [ ] Is my description of issue clear enough for this problem?
+   * If there are some errors occured, we need details about `how do you run your code?`, `what system do you use?`, `Are you using GPU or not?`, etc.
+   * If you could provide an [asciinema](https://asciinema.org/) record, it will be very great and we could help you more efficiently.
```

Comment:
it will be very great and we could help you more efficiently.
-> That's awesome ! We could help you solving the problem you identified more quickly.